src/neuro_flow/batch_runner.py

```diff
@@ -493,6 +505,7 @@ async def _run_bake(
                 f"--tag=flow:{bake.batch}",
                 f"--tag=bake_id:{bake.id}",
                 f"--tag=remote_executor",
+                f"--project={flow._ast_project.project_name}",
```

Comment:
minor. it might make sense to have `flow.project_name` not to dig into implementation details.

Reply:
good idea, added property in [f34f806](https://github.com/neuro-inc/neuro-flow/pull/1081/commits/f34f806f3a0b29dbe0be3ba5d0b76dfce494b381)